src/js/models/ItemList.js

```diff
@@ -0,0 +1,242 @@
+export default class ItemList {
```

Comment:
Yeah, I still think they could be brought together.  In the latest models iteration, I adopted a different design pattern and no longer have this huge corpus object that acts as a controller for the model, instead the overall model is like a figure-8 now.  The Vue.containers access data from the data models and can display them through templating and request modifications through services embedded in the models (so: mc->mv->vc->mc...).  The models take care of syncing with the database (mc->db->mc...).  So these plain JS model/controllers (the "mc") are shared between the Vue component loop and the backend db loop.  I guess this helps mainly with debugging since it provides a single point that manages consistency.  I think you were going in a different direction with a separate controller for your models, maybe that is better, if so, I can change, since it is easier than ever to change things now.